sdk/ai/azopenai/example_client_audio_test.go

```diff
@@ -120,3 +120,49 @@ func ExampleClient_GenerateSpeechFromText() {
 
 	// Output:
 }
+
+func ExampleClient_AudioTranslation() {
+	openAIKey := os.Getenv("OPENAI_API_KEY")
+
+	// Ex: "https://<your-azure-openai-host>.openai.azure.com"
+	azureOpenAIEndpoint := os.Getenv("AZURE_OPENAI_ENDPOINT")
+
+	modelDeploymentID := os.Getenv("AZURE_OPENAI_DEPLOYMENT_ID")
+
+	if openAIKey == "" || azureOpenAIEndpoint == "" || modelDeploymentID == "" {
+		fmt.Fprintf(os.Stderr, "Skipping example, environment variables missing\n")
+		return
+	}
+
+	keyCredential := azcore.NewKeyCredential(openAIKey)
+
+	client, err := azopenai.NewClientWithKeyCredential(azureOpenAIEndpoint, keyCredential, nil)
+
+	if err != nil {
+		// TODO: Update the following line with your application specific error handling logic
+		log.Printf("ERROR: %s", err)
+		return
+	}
+	mp3Bytes, err := os.ReadFile("../../sdk/ai/azopenai/testdata/sampleaudio_hindi_myVoiceIsMyPassportVerifyMe.mp3")
```

Comment:
```suggestion
	mp3Bytes, err := os.ReadFile("testdata/sampleaudio_hindi_myVoiceIsMyPassportVerifyMe.mp3")
```

Go will put you in the same folder as your test, when it runs, so we don't need to use relative paths here.

Reply:
done